packages/docs/docs/tour/reports.md

```
@@ -1,33 +1,24 @@
 ![Cashflow report](/img/a-tour-of-actual/tour-reports-cashflow.webp)
 
 ## Custom reports
 
-Actual custom reports should give you all the reports you need on your spending habits and your income and are
-thoroughly covered in the [Custom Reports topic](/docs/reports/custom-reports) in the manual.
+Actual's custom reports will give you in-depth analysis on your spending habits and your income. You can find them thoroughly covered in [Custom Reports](/docs/reports/custom-reports).
 
-Below are examples of reports covering our *Daily Expenses* over the last six months. One report shows this as a graph,
-the other as a table. Note that the table version also shows the sum and average (over six months)
-for the various categories.
+Below are examples of reports covering *Daily Expenses* over the last six months. One report shows this as a graph, the other as a table. Note that the table version also shows the sum and average (over six months) for the various categories.
```

markdownlint-cli2 (0.22.0)

[warning] Emphasis style
Expected: underscore; Actual: asterisk

(MD049, emphasis-style)